susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1107,9 +1111,9 @@
         'PDID' : [2145, 1712], 'BETAPDID' : [1928], 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15_X86_ARM,
         'DEST' : '/srv/www/htdocs/pub/repositories/sle/15/3/bootstrap/'
     },
-    'SUMA-41-PROXY-x86_64' : {
-        'PDID' : [1946, 2015], 'BETAPDID' : [], 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        'DEST' : '/srv/www/htdocs/pub/repositories/sle/15/2/bootstrap/'
+    'SUMA-42-PROXY-x86_64' : {
+        'PDID' : [2145, 2225], 'BETAPDID' : [], 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15_X86_ARM,
```

Comment:
I wonder if we should have tbe BETAPDID for the beta client tools. @mcalmer?

Otherwise beta users will be using the non-beta client tools.

Reply:
No, not needed. Everything what is required should be in the "Proxy Module".
Proxies do not get any client tools repos.

Reply:
Ah, true :-D